.circleci/config.yml

```diff
@@ -65,6 +65,16 @@ jobs:
       - <<: *restore_dependency_cache
       - run: npm run test:examples
 
+  test_macos:
+    macos:
+      xcode: "10.0"
+    steps:
+      - checkout
+      - run: npm install
```

Comment:
You should be able to just restore the dependency cache here.

```
- checkout
- <<: *restore_dependency_cache
```

Reply:
Since the test is on a different platform, I'm not sure the cache is available to restore.